package.json

```diff
@@ -64,19 +70,23 @@
     "lerna": "^3.13.0",
     "lint-staged": "^8.1.4",
     "prettier": "^1.16.4",
-    "react": "16.9.0-alpha.0",
-    "react-dom": "16.9.0-alpha.0",
-    "react-redux": "^7.0.2",
-    "react-router": "^5.0.0",
-    "react-router-dom": "^5.0.0",
-    "react-test-renderer": "^16.8.6",
-    "redux": "^4.0.1",
     "require-context.macro": "^1.0.4",
     "shx": "^0.3.2",
     "typescript": "~3.4",
     "web-ext": "^3.0.0"
   },
   "dependencies": {
-    "react-scripts": "3.0.0"
+    "@material-ui/core": "4.0.1",
+    "@material-ui/styles": "4.0.1",
+    "final-form": "^4.12.0",
+    "react-final-form-hooks": "^2.0.0",
+    "react": "16.9.0-alpha.0",
+    "react-dom": "16.9.0-alpha.0",
+    "react-redux": "^7.0.2",
+    "react-router": "^5.0.0",
+    "react-router-dom": "^5.0.0",
+    "react-scripts": "3.0.0",
+    "react-test-renderer": "^16.8.6",
+    "redux": "^4.0.1"
```

Comment:
A lerna monorepo should not have any "dependencies", as only developers will ever install those dependencies, not users.

Reply:
🤔 let me understand this. Which can be the use case of users using our repo without being developers? The only one I can think of is the usage of medulas-react-components, and for that.... maybe is just worth use `peerDependencies`?

Basically what I want to do is, have a way to autodeploy the extension to the chrome store #55, and also autodeploy our example webapps to our AWS instance, so, under this use case, does it hurt, under this circumstances, to still have those dependencies here?

Sorry but I am kind of lost. Please help! 😉

Reply:
> Which can be the use case of users using our repo without being developers? 

Such a use case cannot exist. The repo root package.json is only for developers.

> maybe is just worth use peerDependencies

Nope, peerDependencies are something different

> does it hurt, under this circumstances, to still have those dependencies here?

The point is: only developers (and the CI) install the dependencies from repo root package.json. Thus all the dependencies in that file should be in "devDependencies". "dependencies" must be empty.

Reply:
By the way, a "user" for which the "dependencies" field exists, is someone installing a package from npm.

Reply:
Ok, I can move the `dependencies` to `devDepencies` based on the premise of root's package belongs to developers. So far so good.

But I do not really understand why libraries which are needed for running deployed apps should be in `devDependencies`.

Reply:
> But I do not really understand why libraries which are needed for running deployed apps should be in devDependencies.

Good point, see my other comment. If a runtime dependency is in the root package.json, this is an error.